File: 2026-01/src/day_06/boj_10886/Main.java
package day_06.boj_10886;

import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int n = Integer.parseInt(br.readLine());
        int cnt = 0;

        while (n-- > 0) {
            int x = Integer.parseInt(br.readLine());

            if (x == 0) {
                cnt++;
            } else {
                cnt--;
            }
        }

        if (cnt > 0) {
            System.out.println("Junhee is not cute!");
        } else {
            System.out.println("Junhee is cute!");
        }
    }
}

File: 2026-01/src/day_06/boj_10886/Main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int n;
    cin >> n;

    int cnt = 0;

    while (n--) {
        int x;
        cin >> x;

        if (x == 0) {
            cnt++;
        } else {
            cnt--;
        }
    }

    if (cnt > 0) {
        cout << "Junhee is not cute!";
    } else {
        cout << "Junhee is cute!";
    }
}
